src/typedefs.js

```diff
@@ -184,6 +184,33 @@ const React = require('react');
  * @memberof typedefs
  */
 
+/**
+ * clientListTableProps is an object that stores the props for the
+ * ClientListTable component
+ *
+ * @exports clientListTableProps
+ * @typedef {object} clientListTableProps
+ * @property {React.Dispatch<React.SetStateAction<boolean>>} setShowDeleteClientModal
+ * - React set function for DeleteClientModal
+ * @property {React.Dispatch<React.SetStateAction<null>>} setSelectedClientToDelete
+ * - React set function for setting client to be deleted
+ * @memberof typedefs
+ */
+
+/**
+ * clientListTableRowProps is an object that stores the props for the
+ * ClientListTableRow component
+ *
+ * @exports clientListTableRowProps
+ * @typedef {object} clientListTableRowProps
+ * @property {object} client - Object containing client information
+ * @property {React.Dispatch<React.SetStateAction<boolean>>} setShowDeleteClientModal
+ * - React set function for DeleteClientModal
+ * @property {React.Dispatch<React.SetStateAction<null>>} setSelectedClientToDelete
+ * - React set function for setting client to be deleted
+ * @memberof typedefs
+ */
+
```

Comment:
Recent changes in Development moved typedefs for Contacts within ContactListTable.jsx and ContactListTableRow.jsx, so these changes could actually be removed.